Lombiq.NodeJs.Extensions/scripts/lint-markdown.js

```diff
@@ -0,0 +1,117 @@
+const fs = require('fs');
+const markdownlint = require('markdownlint'); // eslint-disable-line import/no-unresolved -- False positive, it's in the package.json.
+const path = require('path');
+const process = require('process');
+const textlint = require('textlint'); // eslint-disable-line import/no-unresolved -- False positive, it's in the package.json.
+const findRecursively = require('./find-recursively');
+
+const markdownlintConfig = JSON.parse(fs.readFileSync(path.resolve(__dirname, '..', 'config', 'lombiq.markdownlint.json')));
```

Comment:
Add the encoding to this `readFileSync()` call, too.

Reply:
I intentionally didn't add the encoding, because then the function returns a different type.
https://nodejs.org/api/fs.html#:~:text=if%20the%20encoding%20option%20is%20specified%20then%20this%20function%20returns%20a%20string.%20otherwise%20it%20returns%20a%20buffer.

Reply:
hmm, the documentation was misleading and I thought it'd behave differently. Just tried it and it still works so I've added it.